apps/dashboard/src/components/workflow-editor/steps/email/maily-config.tsx

```diff
@@ -127,8 +128,23 @@ export const createExtensions = (props: {
           as: 'div',
         });
       },
+      addAttributes() {
+        const attributes = this.parent?.();
+        return {
+          ...attributes,
+          aliasFor: {
+            default: null,
+          },
+        };
+      },
     }).configure({
-      suggestion: getVariableSuggestions(VARIABLE_TRIGGER_CHARACTER),
+      suggestion: {
+        ...getVariableSuggestions(VARIABLE_TRIGGER_CHARACTER),
+        command: ({ editor, range, props }) => {
+          const query = props.id + '}}';
+          insertVariableToEditor({ query, editor, range, isAllowedVariable: parsedVariables.isAllowedVariable });
+        },
+      },
```

Comment:
This executes when you either click on variable from suggestion or press enter mid-typing. It needs to execute the same function as when you type the full variable and close with `}}`.